bioverse/analysis.py

```diff
@@ -131,7 +208,7 @@ def test_hypothesis_grid_iter(h, generator, survey, bins, return_chains,
     results['bins'], results['values'], results['errors'] = bins, values, errors
     results['t_sim'], results['t_fit'] = t_sim, t_fit
     
-    return results
+    return results, log_entries, iter_num
```

Comment:
could this break backward compatibility? (probably not, because the user doesn't usually call this function directly)

Reply:
Gotcha. I adapted the returned variables in `analysis.random_simulation`.